tasks/src/main/java/ru/sberbank/school/task07/Counter.java

```diff
@@ -1,18 +1,18 @@
-package ru.sberbank.school.task07;
-
-
-/**
- * Считатель различных слов в файле.
- * <p>
- * Задание: Подсчитайте количество различных слов в файле.
- */
-public interface Counter {
-
-    /**
-     * Подсчёт различных слов в файле.
-     *
-     * @param pathToFile путь до файла
-     * @return количество различных слов
-     */
-    int count(String pathToFile);
-}
+package ru.sberbank.school.task07;
+
+
+/**
+ * Считатель различных слов в файле.
+ * <p>
```

Comment:
&lt;p&gt; tag should be placed immediately before the first word, with no space after.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a489011200>*